.github/workflows/build-and-deploy.yml

```diff
@@ -137,6 +137,31 @@ jobs:
           environment: test_aks
           azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
 
+  deploy_v2_preprod:
```

Comment:
Can we use a matrix for dev, test, preprod such as [deploy_nonprod](https://github.com/DFE-Digital/find-a-lost-trn/blob/main/.github/workflows/build-and-deploy.yml#L140) ? This would avoid repetition and reduce the size of the workflow.